### ISSUE TITLE
#[help] should not force the definition of the owners hashset with `impl BuildHasher`

### ISSUE THREAD

**Issue by nickelc** [serenity-rs/serenity#570]
The help macro should not force the definition of `owners` as `HashSet<UserId, impl BuildHasher>` while `Configuration::owners` takes `HashSet<UserId>`.

It only makes the code noisy and i think the default hasher should be sufficient.

**Comment by nickelc** [comment]
I've looked into the source to find the reasoning for this requirement and found the clippy warning `implicit_hasher` when i removed `impl BuildHasher` in `help_commands.rs`. 

Then i removed the requirement in `command_attr` and my example builds fine with `owners: HashSet<UserId>` and without clippy warnings for my code.

```
$ cargo clippy --version
clippy 0.0.212 (726176e 2019-04-18)
```

**Comment by Lakelezz** [comment]
While I could not reproduce your steps taken, I removed and allowed the implicit hasher for the sake of simplicity. Should be fixed as of #579.
